### DISCUSSION TITLE
A Open-Source collection of schema templates for pocketbase

### DISCUSSION THREAD

**Discussion by bytekai** [pocketbase/pocketbase#4293] [Show and tell]
So I was recently looking for templates to just spin up a PocketBase instance to show some people how PocketBase works, but I could not find anything online, so I created a repository where I'm going to be putting some templates I think would be useful.

https://github.com/Pocket-Space/pocketbase-templates

The final plan is to allow users to select a template if they wish in the https://pocketspace.dev Interface to avoid having to create their schemas from scratch.

There are probably a bunch of mistakes and stuff that can be improved, so please if you find anything, either make an issue or a PR ❤️

**Comment by bennythemink** [comment]
Great idea! This could be really useful.
